samples/subsys/usb/hid-mouse/sample.yaml

```diff
@@ -0,0 +1,6 @@
+sample:
+  name: USB HID mouse sample
+tests:
+  test_hid:
```

Comment:
There is no button on tinytile board. Try `platform_exclude: tinytile` to avoid building this sample on it.

Reply:
done